### PR TITLE
Document List | New expanded comments styling

### PR DESCRIPTION
**Pull request by dannysellers**
Connects #3167

## AC
- [ ] Verify that comment box has the following in this order
    - Relevant date
    - Page number/ Jump to section link below
    - Comment text
- [ ] Verify that comment box length is now the same width as table row line
- [ ] Comment box should now be padded 25px
- [ ] There should be 45px spacing between relevant date, page number and comment

### CODE REVIEW

File: app/assets/stylesheets/react/_comment.scss [hunk 1]
```diff
@@ -31,10 +35,13 @@
   word-wrap: break-word;
   word-break: break-word;
   background-color: $color-white;
-  border-radius: 3px;
-  padding: 15px;
-  margin-bottom: 15px;
-  border: 1px solid $color-gray-lighter;
+  padding: 25px;
```
Comment (mdbenjam) [inline app/assets/stylesheets/react/_comment.scss]: This made the comments on the document view page really paddingy. Was that intended?

Comment (dannysellers) [reply app/assets/stylesheets/react/_comment.scss]: The spec called for 25px padding around the comment text, and 40px spacing between the date, page number, and comment. I actually just noticed that I haven't quite matched that—the space between the Jump to Section link and the text content is only 25px at the moment (the text's padding)—but I agree.

image

Comment (mdbenjam) [reply app/assets/stylesheets/react/_comment.scss]: Oh I'm not really concerned about that. I was talking about the document view page.
screen shot 2017-12-22 at 6 14 37 pm

Comment (dannysellers) [reply app/assets/stylesheets/react/_comment.scss]: It looks like that padding has always been there—there's a [40px `margin-bottom`](https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/assets/stylesheets/_main.scss?utf8=%E2%9C%93#L683) on `.usa-accordion`. Changing it to 20 does make it a bit better, but that might also be a question for @gnakm 

40px
40px
20px
20px

Comment (gnakm) [reply app/assets/stylesheets/react/_comment.scss]: 20px is better and thank you for asking!

Comment (gnakm) [reply app/assets/stylesheets/react/_comment.scss]: @dannysellers also, current implementation for document list comments padding looks good 👍

Comment (dannysellers) [reply app/assets/stylesheets/react/_comment.scss]: 👍

Comment (mdbenjam) [reply app/assets/stylesheets/react/_comment.scss]: Sorry I keep not describing exactly what I mean on this :)

I was referring to the padding inside the comment. It used to look like this:
screen shot 2018-01-02 at 10 52 15 am

Now it's this:
screen shot 2018-01-02 at 10 52 03 am

I know we want that on the document list page, just not sure if we wanted the same on this page.

File: app/assets/stylesheets/react/_comment.scss [hunk 2]
```diff
@@ -31,10 +35,20 @@
...
-  margin-bottom: 15px;
-  border: 1px solid $color-gray-lighter;
+
+  &:not(.comment-horizontal-container) {
+    margin-bottom: 15px;
+    border-radius: 3px;
+    border: 1px solid $color-gray-lighter;
+
+    &:not(.comment-textarea) {
+      padding-left: 25px;
```
Comment (mdbenjam) [inline app/assets/stylesheets/react/_comment.scss]: Looks like this would've defaulted to 15px from above, rather than 25px just on the left.

**Pull request merged**